rn-cli-inside-gb.config.js

```diff
@@ -1,6 +1,7 @@
 /** @format */
-const blacklist = require( 'metro' ).createBlacklist;
 const path = require( 'path' );
+const blacklist = require( 'metro-config/src/defaults/blacklist' );
+const blacklistElements = blacklist( [ new RegExp( path.basename( __dirname ) + '/gutenberg/.*' ) ] );
```

Comment:
It seems that while moving this blacklisting here, we lost the inline comment that was there before. Let's reinstate the comment, or just perhaps just move the line back as it was, without defining the const here. WDYT @SergioEstevao ?